### ISSUE TITLE
Missing filters in model

### ISSUE THREAD

**Issue by SamGuthrie** [LucasKirsten/keras-portraitnet#1]
Hi, Thanks for putting this together. 

I'm looking at implementing and training this model, however the model fails to run because of this line:
` 

        for (f,k) in filters:

            p = DimReduceBlock(p, f, k, self.dropout)

            concat_layers.append(p)
`
"filters" is never declared.

Also, which backbone is this using - the original paper uses MobileNetV2 however I don't see any references to this in the code?

**Comment by LucasKirsten** [comment]
Hi, you are welcome 👍

I just fixed the network code in the new commit, please refer to it.

In fact, I did not use the MobileNetV2 for the backbone of the model. In the next commits I intend to address this.
